src/types/index.ts

```diff
@@ -1,21 +1,51 @@
-export type Task = {
-  is_deleted: boolean;
-  type?: 'homework';
-};
+export interface ExercisePersonData {
+  user_id: ID;
+  name: string;
+}
 
-type CollaboratorSolution = {
+export interface ExerciseAnswerData {
+  id: ID;
   content_html: string;
-};
+  correctness?: string;
+  feedback_html?: string;
+  isCorrect?: boolean;
+}
+
+export interface ExerciseTagData {
+  id: ID;
+  type: string;
+  is_visible: boolean;
+  data: string;
+}
 
-export type Question = {
+export interface ExerciseQuestionData {
   id: ID;
+  is_answer_order_important: boolean;
+  stimulus_html: string;
   stem_html: string;
-  collaborator_solutions: CollaboratorSolution[];
-  formats?: string[];
+  answers: ExerciseAnswerData[];
+  formats: ExerciseFormat[];
+  collaborator_solutions?: CollaboratorSolution[];
+}
+
+export interface ExerciseData {
```

Comment:
Updated the type names to match Tutor and use the full definitions with only a few exceptions that felt right to leave out.